### PR TITLE
schema: allow blank settings pages

### PR DESCRIPTION
**Pull request by kpsherva**

### CODE REVIEW

File: invenio_communities/communities/schema.py
Comment (kpsherva) [reply invenio_communities/communities/schema.py:116]: the problem is that if you are trying to change any other property (fe. direct publish), not_blank is blocking this action via validation error, but it is confusing for the user, because the about page input is under another tab (different from curation policy), so either we allow the empty values, or we split the validation schemas, or we do some kind of partial validation, or patch for chosen fields only (basically we don't validate them together). 
Let me know which one is the preferred solution (long term), for now I would go with this one in order not to block the release.

Comment (slint) [reply invenio_communities/communities/schema.py:116]: Ok, I agree on the solution, so that we unblock this. As long as the UI plays well, we're good.

---

Long-term IIRC, my take on the REST API was that we should always support PATCH-style behavior (even for PUT requests), i.e.:

1. If a field is skipped, it's not stored/updated, and is left as-is.
2. If a field is set to `None`, it means that its value is "cleared".
3. `1.` and `2.` apply only to top-level fields of a resource (this simplifies things for lists of objects and generally deeply-nested data).
4. In the response the full resource is returned/serialized (i.e. even fields that were not touched)

(Probably have make a tattoo of this to remember in the future 🙈 )

Comment (kpsherva) [reply invenio_communities/communities/schema.py:116]: 👍 for the tattoo

**Review by jennur — APPROVED** [review]
LGTM! Thanks for fixing this 🙏👏

**Pull request merged**
